lib/ansible/modules/cloud/kubevirt/kubevirt_pvc.py

```diff
@@ -0,0 +1,279 @@
+    - inline
+    type: dict
+  state:
+    description:
+    - Determines if an object should be created, patched, or deleted. When set to C(present), an object will be
+      created, if it does not already exist. If set to C(absent), an existing object will be deleted. If set to
+      C(present), an existing object will be patched, if its attributes differ from those specified using
+      module options and I(resource_definition).
+    default: present
+    choices:
+    - present
+    - absent
+  force:
+    description:
+    - If set to C(True), and I(state) is C(present), an existing object will be replaced.
+    default: false
+    type: bool
+  merge_type:
+    description:
+    - Whether to override the default patch merge approach with a specific type.
+    - "This defaults to C(['strategic-merge', 'merge']), which is ideal for using the same parameters
+      on resource kinds that combine Custom Resources and built-in resources."
+    - See U(https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/#use-a-json-merge-patch-to-update-a-deployment)
+    - If more than one merge_type is given, the merge_types will be tried in order
+    choices:
+    - json
+    - merge
+    - strategic-merge
+    type: list
+  name:
+    description:
+      - Use to specify a PVC object name.
+    required: true
+    type: str
+  namespace:
+    description:
+      - Use to specify a PVC object namespace.
+    required: true
+    type: str
+  annotations:
+    description:
+      - Annotations attached to this object.
+      - U(https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+    type: dict
+  labels:
+    description:
+      - Labels attached to this object.
+      - U(https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+    type: dict
+  selector:
+    description:
+      - A label query over volumes to consider for binding.
+      - U(https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+    type: dict
+  access_modes:
+    description:
+      - Contains the desired access modes the volume should have.
+      - More info: U(https://kubernetes.io/docs/concepts/storage/persistent-volumes#access-modes)
+    type: list
+  size:
+    description:
+      - How much storage to allocate to the PVC.
+    type: str
+    aliases:
+    - storage
+  storage_class_name:
+    description:
+      - Name of the StorageClass required by the claim.
+      - More info: U(https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1)
+    type: str
+  volume_mode:
+    description:
+      - volumeMode defines what type of volume is required by the claim. Value of Filesystem is implied when not included in claim spec. This is an alpha feature of kubernetes and may change in the future.
+    type: str
+  volume_name:
+    description:
+      - volumeName is the binding reference to the PersistentVolume backing this claim.
+    type: str
+  content_type:
```

Comment:
It seems like `content_type` is related to `source` so why it is outside?

Reply:
That's the structure of datavol yaml, but in the context of the this module it makes sense to hide it. So I've moved it inside `cdi_source`.